runtime/doc/api.txt

```diff
@@ -2527,6 +2527,7 @@ nvim_buf_get_extmark_by_id({buffer}, {ns_id}, {id}, {opts})
       • {id}      Extmark id
       • {opts}    Optional parameters. Keys:
                   • details: Whether to include the details dict
+                  • verbose: Whether to include verbose details
```

Comment:
I would expect verbose to output both. Maybe rename to `names` and document that it defaults to true.

Reply:
Yeah I named it `verbose` in case we want to group other verbose details under the same key in the future. If granularity is preferred instead I can rename it.

Reply:
Details is the effective `verbose`, so I think a rename is best.